[2025-05-25T05:14:35.328+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-25T05:14:36.828+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: load_file_to_psql.generate_file scheduled__2024-11-29T00:00:00+00:00 [queued]>
[2025-05-25T05:14:37.743+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: load_file_to_psql.generate_file scheduled__2024-11-29T00:00:00+00:00 [queued]>
[2025-05-25T05:14:37.758+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-05-25T05:14:38.932+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): generate_file> on 2024-11-29 00:00:00+00:00
[2025-05-25T05:14:39.638+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=190) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-05-25T05:14:39.767+0000] {standard_task_runner.py:72} INFO - Started process 236 to run task
[2025-05-25T05:14:39.939+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'load_file_to_psql', 'generate_file', 'scheduled__2024-11-29T00:00:00+00:00', '--job-id', '17', '--raw', '--subdir', 'DAGS_FOLDER/data_quality.py', '--cfg-path', '/tmp/tmpy7bjz1ps']
[2025-05-25T05:14:40.608+0000] {standard_task_runner.py:105} INFO - Job 17: Subtask generate_file
[2025-05-25T05:15:01.078+0000] {job.py:229} INFO - Heartbeat recovered after 30.34 seconds
[2025-05-25T05:15:01.596+0000] {task_command.py:467} INFO - Running <TaskInstance: load_file_to_psql.generate_file scheduled__2024-11-29T00:00:00+00:00 [running]> on host b4cf172ed0b9
[2025-05-25T05:15:06.299+0000] {job.py:229} INFO - Heartbeat recovered after 19.92 seconds
[2025-05-25T05:15:06.836+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='https://t.me/Shust_DE' AIRFLOW_CTX_DAG_OWNER='@Shust_DE' AIRFLOW_CTX_DAG_ID='load_file_to_psql' AIRFLOW_CTX_TASK_ID='generate_file' AIRFLOW_CTX_EXECUTION_DATE='2024-11-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-29T00:00:00+00:00'
[2025-05-25T05:15:06.867+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-05-25T05:15:10.411+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-05-25T05:15:12.718+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-05-25T05:15:12.771+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=load_file_to_psql, task_id=generate_file, run_id=scheduled__2024-11-29T00:00:00+00:00, execution_date=20241129T000000, start_date=20250525T051436, end_date=20250525T051512
[2025-05-25T05:15:14.994+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-25T05:15:15.281+0000] {standard_task_runner.py:217} INFO - Process not found (most likely exited), stop collecting metrics
[2025-05-25T05:15:15.567+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
